pyne/fispact.py

```diff
@@ -200,9 +200,11 @@ def check_fisp_version(data):
 
 
 def isFisII(data):
-    """boolean check if file is fispact-ii output """
-    v = check_fisp_version(data)
-    if v == "FISPACT-II":
+    """boolean check if file is fispact-ii output (or fispact output for the program user's convinience)"""
+    v = check_fisp_version(data) 
+    if v == "FISPACT":
```

Comment:
A test is needed to check this case. Can you add a test that demonstrates the FISPACT (rather than FISPACT-II) behavior here?